resources/code/lf-emr-workshop-js.template

```diff
@@ -0,0 +1,721 @@
+          }
+        ]
+      }
+    },
+    "LFEMRSecurityConfiguration": {
+      "Type": "AWS::EMR::SecurityConfiguration",
+      "DependsOn": [
+        "LakeFormationSAMLRole"
+      ],
+      "Properties": {
+        "Name": "LF-EMR-SecurityConfiguration",
+        "SecurityConfiguration": {
+          "LakeFormationConfiguration": {
+            "IdpMetadataS3Path": {
+              "Ref": "IdentityProviderMetaDataInS3"
+            },
+            "EmrRoleForUsersARN": {
+              "Fn::GetAtt": [
+                "LakeFormationEMRRole",
+                "Arn"
+              ]
+            },
+            "LakeFormationRoleForSAMLPrincipalARN": {
+              "Fn::GetAtt": [
+                "LakeFormationSAMLRole",
+                "Arn"
+              ]
+            }
+          },
+          "AuthenticationConfiguration": {
+            "KerberosConfiguration": {
+              "Provider": "ClusterDedicatedKdc",
+              "ClusterDedicatedKdcConfiguration": {
+                "TicketLifetimeInHours": 24
+              }
+            }
+          }
+        }
+      }
+    },
+    "S3EMRLogsBucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": {
+          "Fn::Join": [
+            "-",
+            [
+              "emr-logs",
+              {
+                "Ref": "AWS::AccountId"
+              }
+            ]
+          ]
+        }
+      }
+    },"S3NotebooksBucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": {
+          "Fn::Join": [
+            "-",
+            [
+              "notebooks",
+              {
+                "Ref": "AWS::AccountId"
+              }
+            ]
+          ]
+        }
+      }
+    },
+    "EMRCluster": {
+      "Type": "AWS::EMR::Cluster",
+      "DependsOn": [
+        "LFEMRSecurityConfiguration",
+        "LakeFormationEMRInstanceProfileRole",
+        "LFEMRInstanceProfile",
+        "S3NotebooksBucket"
+      ],
+      "Properties": {
+        "Instances": {
+          "MasterInstanceGroup": {
+            "InstanceCount": 1,
+            "InstanceType": {
+              "Ref": "InstanceType"
+            },
+            "Market": "ON_DEMAND",
+            "Name": "cfnMaster"
+          },
+          "CoreInstanceGroup": {
+            "InstanceCount": 1,
+            "InstanceType": {
+              "Ref": "InstanceType"
+            },
+            "Market": "ON_DEMAND",
+            "Name": "cfnCore"
+          },
+          "Ec2SubnetId": {
+            "Ref": "VPCPublicSubnet"
+          },
+          "Ec2KeyName": {
+            "Ref": "EC2KeyPair"
+          }
+        },
+        "Name": "LF-EMRCluster",
+        "Applications": [
+          {
+            "Name": "Hadoop"
+          },
+          {
+            "Name": "Spark"
+          },
+          {
+            "Name": "Livy"
+          },
+          {
+            "Name": "Zeppelin"
```

Comment:
Should we include Hive ? I know Hive and Presto do not support for LF yet but it might be easier once the support becomes available. Presto might have to be a different cluster but Hive and Spark can be on the same cluster

Reply:
We can include Hive and Presto later when the functionality is available to reduce the time it takes now.

Reply:
Let's add it to list of ToDos